### ISSUE TITLE
List bug

### ISSUE THREAD

**Issue by phlind** [MalignantCarp/obsidian-typography-plugin#7]
Hey, I found a bug that the plugin doesn't work if the quote is the first item in a list that has a sub-item below it. For reproducing it, you can just paste that into Obsidian:

```
- "Something in quotation marks"
- "Something something"
	- "Something else"
```

On the second line it won't work.

**Comment by MalignantCarp** [comment]
Interestingly, if the list item follows a regular paragraph, it seems to be correctly styled (though the second list item is not).

![image](https://user-images.githubusercontent.com/93364935/146688488-def232c7-1412-4b6c-b4a2-b57af1d2ac12.png)

I imagine this all revolves around how I bypass styling HTML tags. I suspect that is likewise the cause of #8, so I may need to work on a new means of bypassing HTML tags and the like a little sooner than I had expected.

**Comment by MalignantCarp** [comment]
This should be fixed in 0.0.6. Please let me know if it is resolved for you and I will close this issue.
